client.go

```diff
@@ -1475,4 +1484,9 @@ type baseCommit struct {
 type baseCommitPartial struct {
     Dan string `xml:"device-and-network,omitempty"`
     Pao string `xml:"policy-and-objects,omitempty"`
+    Admin *baseCommitPartialAdmin `xml:"admin,omitempty"`
```

Comment:
If the admin listing is just a list of members, then use `util.MemberType` instead.  There is also the helpful `util.StrToMem()` that you can use.  So do this instead:

```golang
Admin *util.MemberType `xml:"admin"`
```

And finally, you can remove the `,omitempty` from this as pointers to structs are helpfully omitted if the pointers are `nil`.

Reply:
Thanks for your feedback, this change is cleanly!